hpx/lcos/local/recursive_mutex.hpp

```diff
@@ -176,6 +180,8 @@ namespace hpx { namespace lcos { namespace local
             {
                 if (mtx.try_lock())
                 {
+                    util::ignore_lock(&mtx);
+                    //util::register_lock(this);
```

Comment:
AFAICS, you have it commented out: https://github.com/STEllAR-GROUP/hpx/blob/parcelport_plugins/hpx/lcos/local/recursive_mutex.hpp#L158